dynamiqs/integrators/apis/dssesolve.py

```diff
@@ -28,18 +28,21 @@ def dssesolve(
         This function has not been implemented yet. The following API is indicative
         of the planned implementation.
 
+    In quantum optics, the diffusive SSE describes the evolution of a quantum
+    system under homodyne or heterodyne detection with unit detection efficiency.
     This function computes the evolution of the state vector $\ket{\psi(t)}$ at time $t$,
     starting from an initial state $\ket{\psi_0}$ according to the diffusive SSE in Itô
-    form (with $\hbar=1$ and where time is implicit(1))
+    form ($\hbar=1$, time is implicit(1))
     $$
         \begin{split}
-            \dd\ket\psi = \Bigg[
+            \dd\\!\ket\psi = \Bigg[
                 &-iH \dt
                 -\frac12 \sum_{k=1}^N \left(
-                    L_k^\dag L_k - \braket{\psi| L_k + L_k^\dag |\psi} L_k + \braket{\psi| L_k + L_k^\dag |\psi}^2 I
+                    L_k^\dag L_k - \braket{L_k + L_k^\dag} L_k
+                    + \braket{L_k + L_k^\dag }^2 \\! / 4
```

Comment:
I think we should put the back `I`.

Reply:
Why? There is no ambiguity here, I'm very much in favor of keeping the equation as simple and readable as possible.

Reply:
Many people (including myself and Gaspard for example) bug when mixing matrices with numbers (much like when you can do "a + 2" in QuTip), it's "inhomogeneous". I think it's a notational shortcut that doesn't gain much space and makes the equation harder to understand. But if you strongly prefer this it's fine.

Reply:
I don't quite agree. The equation is perfectly valid here, you're multiplying a constant with a ket, there is no need to introduce an identity operator in between. IMO this additional identity is making the equation more complicated and adds nothing in terms of understanding.

Your example is quite relevant, we always write `a^2 - alpha^2` and not `a^2 - alpha^2 * I` in physics equations, otherwise we start having identity operators all around the place.

Reply:
Ok, as you prefer :)